src/cmds/main.go

```diff
@@ -28,7 +28,7 @@ import (
 func main() {
 
 	cfg := pbft.Config{}
-	cfg.HostsFile = path.Join(os.Getenv("HOME"), "hosts") // TODO: read from config.yaml in future.
+	cfg.HostsFile = path.Join(os.Getenv("PWD"), "hosts") // TODO: read from config.yaml in future.
```

Comment:
hey, change this just like you did here? https://github.com/truechain/truechain-consensus-core/pull/14/files#diff-7bde18e8151279ab2e05cde5897db6f8R1220 
I could be running from anywhere, conf resides in /etc/truechain/ (or something like that as default) and then change-able.

Reply:
It can be ran from anywhere, `log_folder` is just one of the field in the `cnf.ini` file. The path defined on `cfg.ini` can be anything like
```ini
[keys]
pem_keys = /tmp/truechain/keys
[log]
logs = /tmp/truechain/logs
``` 
or 
```ini
[keys]
pem_keys = /etc/truechain/keys
[log]
logs = /etc/truechain/logs
```